lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -67,12 +79,13 @@ defmodule Membrane.MP4.Muxer.CMAF do
       track: nil,
       elapsed_time: 0,
       end_timestamp: 0,
-      buffer_awaiting_duration: nil
+      buffer_awaiting_duration: nil,
+      parts_duration: Membrane.Time.seconds(0)
```

Comment:
The queue keeps only samples for the following partial segment. This field is responsible for keeping the accumulated durations sum of partial segments so far. So if you created 3 partial segments of 0,5s then this will contain 1,5s. Once encountered a keyframe the `parts_duration` gets reset.

It is basically used for different collection strategies (if we are far below the min segment duration then we simply push the sample without doing any fancy checking).